src/databricks/labs/lakebridge/cli.py

```diff
@@ -240,6 +242,14 @@ def use_transpiler_config_path(self, transpiler_config_path: str | None) -> None
             )
             self._config = dataclasses.replace(self._config, transpiler_config_path=transpiler_config_path)
 
+        # Switch is installed inside "/Users/<>/.lakebridge/transpilers/Switch/lsp/config.yml
+        if (
+            self._config.transpiler_config_path is not None
+            and Path(self._config.transpiler_config_path).parent.parent.name == "Switch"
+        ):
+            msg = "Switch transpiler is not supported through `transpile` run `llm-transpile` instead."
+            raise RuntimeError(msg)
+
```

Comment:
I don't think this is needed: we're no longer setting up Switch within the transpile configuration.

Reply:
addressed